datalad/support/annexrepo.py

```diff
@@ -335,3 +336,103 @@ def annex_add_to_git(self, files):
             self.cmd_call_wrapper.run(cmd_list, cwd=self.path)
         else:
             self.git_add(files)
+
+    def annex_initremote(self, name, options):
+        """Creates a new special remote
+
+        Parameters:
+        -----------
+        name: str
+            name of the special remote
```

Comment:
Because Dataset is derived from AnnexRepo, which is derived from GitRepo.
I don't want to accidentially overwrite inherited methods. Especially since there are commands like "add", which have a different meaning in each of this classes and may need the inherited equivalent. I could just use the prefix for those, which would collide, but
a) I find this confusing and inconsistent, so development is easier that way, I think and
b) at the end Dataset should be used anyway, where there are no such prefixes.

Could override and use super, of course, but I think it's more confusing.